tools/src/main/scala/scala/scalanative/codegen/CodeGen.scala

```diff
@@ -345,6 +374,9 @@ object CodeGen {
 
       genBlockHeader()
       indent()
+      if (platform.isWindows && block.pred.isEmpty) {
+        ehVarPosition = currentPosition
```

Comment:
Why do you need buffer's current position here? This looks like a hack.

Reply:
I need to insert a variable on the stack `alloca` for exception handler (this is the only way to receive exception address from catchpad), so I need a something after header but I want to know ahead of time that this scope need that variable (to not add it to every scope). So, it's a hack and I am looking for better solution, any ideas?